File: Intiri.Client/src/app/DTOs/In/register-in.dto.ts

export class RegisterInDTO {
    firstName: string;
    lastName: string;
    countryCode: string;
    phoneNumber: string;

    constructor(
        firstName: string,
        lastName: string,
        countryCode: string,
        phoneNumber: string
    ) {
        this.firstName = firstName;
        this.lastName = lastName;
        this.countryCode = countryCode;
        this.phoneNumber = phoneNumber;
    }
}

File: Intiri.API/Intiri.API/Models/DTO/InputDTO/RegisterInDTO.cs
using System.ComponentModel.DataAnnotations;

namespace Intiri.API.Models.DTO.InputDTO
{
    public class RegisterInDTO
    {
        [Required]
        public string FirstName { get; set; }

        [Required]
        public string LastName { get; set; }

        [Required]
        public string CountryCode { get; set; }

        [Required]
        public string PhoneNumber { get; set; }
    }
}
